google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/AppEngineAssetProvider.java

```diff
@@ -34,10 +37,19 @@ public static AppEngineAssetProvider getInstance() {
   }
 
   /**
-   * Returns the project's appengine-web.xml or null if it does not exist.
+   * Returns the project's appengine-web.xml or null if it does not exist. If there are multiple
+   * appengine-web.xml's then it will return one of them with preference given to those under the
+   * WEB-INF directory.
    */
   @Nullable
   public abstract XmlFile loadAppEngineStandardWebXml(@NotNull Project project,
       @NotNull Artifact artifact);
 
+  /**
+   * @see AppEngineAssetProvider#loadAppEngineStandardWebXml(Project, Artifact). Loads the
```

Comment:
Put the 'Loads the configuration..' As the method level comment above the @see javadoc.